erigon-lib/recsplit/eliasfano32/elias_fano.go

```diff
@@ -27,6 +26,7 @@ import (
 	"unsafe"
 
 	"github.com/erigontech/erigon-lib/common/bitutil"
+	"github.com/erigontech/erigon-lib/common/errors"
```

Comment:
this package name conflicts with stdlib 
maybe this err must be part of `streams` package?

Reply:
I can move to streams. BTW, there exists an error package in erigon-lib already, that’s why I put it in there

Reply:
something new. didn't know

Reply:
moved it to stream package.

Reply:
> something new. didn't know

ah, it came from this PR: https://github.com/erigontech/erigon/pull/11996